source/develop/index.md

```diff
@@ -42,8 +42,6 @@ Several presentations both on the technical side and on the user side are availa
 
 Within oVirt project several teams are taking care of different aspects of the system.
 
-The [oVirt GitHub issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Aovirt+archived%3Afalse) are labeled accordingly to identify the main impacted area.
-
```

Comment:
This link doesn't work if you're not logged in in GitHub, this one seems to work better: <https://github.com/search?q=org%3AoVirt+type%3Aissue+state%3Aopen+&type=issues>
If this one works for you as well I would consider fixing above link rather than delete it.

Reply:
Agreed - fixed